docs/application/native/guides/machine-learning/machine-learning-training.md

```diff
@@ -61,7 +61,10 @@ In the following figure, `data` represents input data or feature and `label` is
 
 ![model](./media/model.png)
 
-Currently, only a sequential neural network is supported:
+Only a sequential neural network is supported (until Tizen 6.0).
```

Comment:
Shouldnt this line be removed as it is mentioned in note now?

Reply:
Right. removed